modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FrontendJSCustomDialog.testcase

```diff
@@ -0,0 +1,58 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property custom.properties = "upgrade.database.auto.run=false";
+	property osgi.modules.includes = "frontend-data-set-sample-web";
+	property portal.acceptance = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Frontend Dataset";
+	property testray.main.component.name = "Frontend Dataset";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given: Enabled custom Dialog") {
+			PortalSettings.openInstanceSettingsAdmin();
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Custom Dialogs",
+				configurationName = "Custom Dialogs",
+				configurationScope = "Virtual Instance Scope");
+
+			SystemSettings.configureSystemSetting(
+				enableSetting = "true",
+				settingFieldName = "Enabled");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			SystemSettings.configureSystemSetting(
+				enableSetting = "false",
+				settingFieldName = "Enabled");
+		}
+	}
+
+	@description = "LPS-165262"
```

Comment:
Add more text to the description:
`This test validates that the liferay custom alert modal dialog can render.`

Reply:
done!